### DISCUSSION TITLE
Short aliases for subcommands of brew

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3154] [Everyday usage]
Hi, I run Fedora on my machine, which has `dnf` as its package manager. Dnf has short aliases for many of its subcommands. Like `in` for `install`, `up` for `update`, `se` for `search` etc.

It would be nice to have this in `brew` too.

**Comment by Bo98** [comment]
`brew up` already exists and there is `brew -S` for `brew search`, but in general you might find https://github.com/Homebrew/homebrew-aliases useful which allows you to create your own aliases the way you'd like them.
